src/test/java/insat/company/platform/web/rest/AccountResourceIntTest.java

```diff
@@ -183,6 +186,8 @@ public void testRegisterValid() throws Exception {
     @Transactional
     public void testRegisterInvalidLogin() throws Exception {
         ManagedUserVM invalidUser = new ManagedUserVM();
+        invalidUser.setSection("GL");
+        invalidUser.setYear("3");
         invalidUser.setLogin("funky-log!n");// <-- invalid
         invalidUser.setPassword("password");
```

Comment:
You have not asserted that the field has been added to the DB or not,
and you have not added a test when the field is already existing in the DB